code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -83,6 +89,20 @@
 		add_blood(blood_color)
 	return
 
+/obj/item/weapon/melee/telebaton/proc/stun(mob/living/target, mob/living/user)
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
```

Comment:
stun proc already requires an ishuman check, just make the target arg `mob/living/carbon/human/target`